### PR TITLE
Support setting default TTL for downstream messages

### PR DESCRIPTION
**Pull request by sophokles73**
This PR extends the concept of *default properties* to the tenant level and then uses this concept to support setting a default *time-to-live* on downstream messages.

### CODE REVIEW

File: core/src/main/java/org/eclipse/hono/util/MessageHelper.java [hunk 1]
```diff
@@ -104,6 +104,27 @@
      */
     public static final String APP_PROPERTY_TENANT_ID = "tenant_id";
 
+    /**
```
Comment (dejanb) [inline core/src/main/java/org/eclipse/hono/util/MessageHelper.java]: Should we add somewhere in the docs what properties can be set as defaults?

Comment (sophokles73) [reply core/src/main/java/org/eclipse/hono/util/MessageHelper.java]: I tried to do this in the protocol adapter user guides, e.g. https://www.eclipse.org/hono/user-guide/http-adapter/#downstream-meta-data ...

Comment (dejanb) [reply core/src/main/java/org/eclipse/hono/util/MessageHelper.java]: OK, now I see it. So only ttl and content-type can be set.

Comment (sophokles73) [reply core/src/main/java/org/eclipse/hono/util/MessageHelper.java]: for the time being: yes. For most of the AMQP defined properties, a default value doesn't make much sense, or are there others you would like to support?

Comment (dejanb) [reply core/src/main/java/org/eclipse/hono/util/MessageHelper.java]: No, it's all good. I just wanted to understand it.

**Pull request merged**